crates/experimental-api/src/gen.rs

```diff
@@ -2137,7 +2160,7 @@ pub mod gen {
         pub transaction_hash: Option<Felt>,
     }
 
-    #[derive(Clone, Debug, Deserialize, Serialize)]
+    #[derive(Clone, Debug, Deserialize, Serialize, Default)]
```

Comment:
What exactly does `Default` for `ContractData` mean? What such a contract can do?

Reply:
Removed Default